spec/druid_tools_spec.rb

```diff
@@ -312,6 +330,7 @@
     let(:workspace) { Dir.mktmpdir }
     let(:dr1) { described_class.new(druid_str, workspace) }
     let(:dr2) { described_class.new(strictly_valid_druid_str, workspace) }
+    let(:dr3) { described_class.new(access_druid_str, workspace) }
```

Comment:
actually, should this be a `DruidTools::AccessDruid`?

Reply:
Yep. Making this and the other changes above now. Thank you!

Reply:
@jmartin-sul This is ready for another look.